src/game/models/mod.rs

```diff
@@ -36,6 +36,16 @@ pub trait Positioned {
     }
 
     fn pos(&self) -> vector::Vector;
+
+    fn dx(&self) -> f64 {
```

Comment:
Do you think we should rename this from `Positioned` now that we are modeling really a moving object? "`Movable`"?